### DISCUSSION TITLE
The following OBS plugins failed to load:  obs-websocket  Please update or remove these plugins.

### DISCUSSION THREAD

**Discussion by Olivotec** [Palakis/obs-websocket#1027] [Q&A] [answered]
Hi there, I used to use the instant replay function on my livestreams and it was awesome! But for some reason, after OBS updated to 28.0.2 (64bits) it doens't seem to work anymore. When I open obs i get this error saying "The following OBS plugins failed to load:  obs-websocket  Please update or remove these plugins." Also, I can't see the OBS socket window on tool anymore to config again.

Can someone help me fix this?
![Screenshot_73](https://user-images.githubusercontent.com/114347780/192148721-5d9115db-b197-47cb-a354-aec4f3cc2abb.png)
![Screenshot_74](https://user-images.githubusercontent.com/114347780/192148720-7dfeb2e7-23c7-4bbc-87d9-3c685e2313f0.png)
![Screenshot_75](https://user-images.githubusercontent.com/114347780/192148718-9cdf3291-a605-4e8a-b2e8-7e019d6371d9.png)

**Comment by cutebunnyyounow** [comment]
me paso lo mismo..

**Comment by tt2468** [comment, marked as answer]
obs-websocket is included with OBS Studio as of 28.0.0. The download for 5.0.1 is marked as only for V27, as it uses Qt5 and is not compatible with OBS 28. Use the `Help -> Check for Integrity` function in OBS to repair your installation.
